src/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java

```diff
@@ -0,0 +1,31 @@
+package org.usfirst.frc4904.robot.humaninterface.operators;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.IndexerGrabberClasp;
+import org.usfirst.frc4904.robot.commands.IntakeRollersIntake;
+import org.usfirst.frc4904.robot.commands.IntakeRollersOuttake;
+import org.usfirst.frc4904.robot.commands.IntakeSquared;
+import org.usfirst.frc4904.robot.commands.OuttakeSquared;
+import org.usfirst.frc4904.standard.humaninput.Operator;
+
+public class DefaultOperator extends Operator {
+	public DefaultOperator() {
+		super("DefaultOperator");
+	}
+
+	public DefaultOperator(String name) {
+		super(name);
+		// TODO Auto-generated constructor stub
+	}
+
+	@Override
+	public void bindCommands() {
+		RobotMap.Component.joystick.button3.onlyWhileHeld(new IntakeSquared());
+		RobotMap.Component.joystick.button3.whenReleased(new IndexerGrabberClasp());
```

Comment:
This feels janky. Is there any way you can set the default state by clasping?

Reply:
We can't make it part of the default command because it might act weirdly if we tell it to go to a certain state repeatedly

Reply:
I just looked @ what we did for GearIO last year, which you will want to model. Essentially we have two commands mapped to button3, one of them being `button3.whenPressed(command to change pistons`, which prevents the solenoids from continuously firing while held. Make sense?

Reply:
That is what this code is doing now